components/renku_data_services/project/models.py

```diff
@@ -63,6 +64,8 @@ class ProjectPatch:
     description: str | None
     keywords: list[str] | None
     documentation: str | None
+    template_id: str | None
```

Comment:
suggestion: I wonder if we can type it as `Literal[""]|None` to make it clearer what can/can't be set.

Reply:
Good point! I've tried to do the same in the API spec but the generated model has an issue. I've filed a bug report for it: https://github.com/koxudaxi/datamodel-code-generator/issues/2191